### ISSUE TITLE
All files marked as executable

### ISSUE THREAD

**Issue by haf** [fable-elmish/elmish#102]
### Description

Follow the tutorial and then run `dotnet restore` in the created project folder.

Now all non-executable files have file mode 744, but they should have 644.

```
-rwxr--r--   1 h  staff   361B Jun 16 15:14 README.md
```

**Comment by et1975** [comment]
Interesting... which tutorial was it?

**Comment by haf** [comment]
```
dotnet new -i "Fable.Template.Elmish.React::*"
dotnet new fable-elmish-react -n Sample
cd Sample
dotnet restore
```

**Comment by et1975** [comment]
Ah, this would be in [templates](/fable-elmish/templates) then...
cc @MangelMaxime

**Comment by MangelMaxime** [comment]
Can you tell me your OS ? 
And the dotnet version ?

I just tested with `2.0.0-preview1-005977` under Windows 10 and I got no problem (all file are with 644)

Do they have the right permission before `dotnet restore` ?

**Comment by et1975** [comment]
I was able to reproduce it with dotnet 1.0.4 on OSX and ubuntu.

**Comment by et1975** [comment]
https://github.com/dotnet/templating/issues/240

**Comment by MangelMaxime** [comment]
Seems like we need to use a problem with `dotnet1.x` and non Windows OS.

**Comment by et1975** [comment]
Bug in the CLI, not worth fighting it since they have fixed it for 2.0.
